curriculum/utils.js

```diff
@@ -91,7 +91,7 @@ const directoryToSuperblock = {
 
 function getSuperBlockFromDir(dir) {
   const superBlock = directoryToSuperblock[dir];
-  if (!superBlock) throw Error(`${dir} does not map to a superblock`);
+  if (!superBlock) return null;
```

Comment:
Rather than ignoring unknown superblocks, can we keep throwing and not visit `.DS_Store` instead? That way it will still let us know if it encounters something weird.

It should be possible to configure `readDirP` to ignore `.DS_Store`.

Reply:
Yep will do.

Reply:
@ojeytonwilliams This will still fail the tests because `.DS_Store` will still be checked.

Reply:
The tests will catch any stray superblocks:
```typescript
    const superBlockPaths = directories.filter(
      x => x !== '00-certifications' && x !== '.DS_Store'
    );
```

Reply:
`.DS_Store` is a file and not a directory. This has tripped me in the past as the script walks through every file and directory so I've just regularly wiped them all across the project.

Reply:
I propose blocking development on MacOS. 🤣

Reply:
> `.DS_Store` is a file and not a directory. This has tripped me in the past as the script walks through every file and directory so I've just regularly wiped them all across the project.

Ah, okay. I still think ignoring it is the way to go. File or directory doesn't matter much!

Reply:
I am fine with closing this.

Reply:
If local development on Mac basically works (except for this), then I skipping the `.DS_Store` file while building seems like the way to go.